### ISSUE TITLE
Feedback on mini_ex8 by Thomas Matthiesen

### ISSUE THREAD

**Issue by ThomasMatthiesen** [ubiquitousman/mini-ex#16]
When I opened the program I saw a search-bar and some kind of slider-bar. I typed in something and pressed the search button and the page was filled with GIFs that related to my search-word. It took my some time to figure out the the input-slider-bar controlled the number of GIFs coming to show on the page.
A fun and useful program that can help you find the GIFs you want, and it looks like you have really done this in a very effective way code-wise.
I saw that you have added a keyPressed function, and I guess that it means that the ENTER key can be used to launch the search - even though this did not work when I used the program. Maybe you could have made it clear what this function does? Or if it does anything at all.

**Comment by ubiquitousman** [comment]
You're right. The keyPressed function makes it possible for the user to click on ENTER to search. It worked perfectly before, but for some reason when having the program run by rawgit, the feature doesn't work. However, if we go and check the program locally, it works without any issues.
